From: Thomas Renninger <trenn@suse.de>
Subject: acpi_os_wait_semaphore(): don't complain about timeout
Patch-mainline: yes/soon
References: 118958

The ASL Acquire operator (17.5.1 in ACPI 3.0 spec) is allowed to
time out and return True without acquiring the semaphore.  There's
no indication in the spec that this is an actual error, so this
message should be debug-only, as the message for successful acquisition
is.

Signed-off-by: Bjorn Helgaas <bjorn.helgaas@hp.com>

Index: work-mm5/drivers/acpi/osl.c
===================================================================
--- work-mm5.orig/drivers/acpi/osl.c	2006-03-23 10:22:40.000000000 -0700
+++ work-mm5/drivers/acpi/osl.c	2006-03-31 14:40:44.000000000 -0700
@@ -816,7 +816,7 @@
 	}
 
 	if (ACPI_FAILURE(status)) {
-		ACPI_EXCEPTION((AE_INFO, status,
+		ACPI_DEBUG_PRINT((ACPI_DB_MUTEX,
 				  "Failed to acquire semaphore[%p|%d|%d], %s",
 				  handle, units, timeout,
 				  acpi_format_exception(status)));
